src/equations/ideal_glm_mhd_multiion.jl

```diff
@@ -471,4 +487,172 @@ end
 
     return dissipation
 end
+
+@doc raw"""
+    source_terms_collision_ion_ion(u, x, t,
+                                   equations::AbstractIdealGlmMhdMultiIonEquations)
+
+Compute the ion-ion collision source terms for the momentum and energy equations of each ion species as
+```math
+\begin{aligned}
+  \vec{s}_{\rho_k \vec{v}_k} =&  \rho_k\sum_{k'}\bar{\nu}_{kk'}(\vec{v}_{k'} - \vec{v}_k),\\
```

Comment:
I assume you have `k'` for consistency with the paper. Alternatively, one could maybe think about using `l` as in the implementation?